### DISCUSSION TITLE
Uncaught TypeError: (0 , _jsxDevRuntime).jsxDEV is not a function

### DISCUSSION THREAD

**Discussion by KKrisu** [swc-project/swc#4149] [Q&A] [answered]
Hello!

I'm switching my react app into SWC compiler and I'm experiencing an issue that I can't find a solution for.
The app compiles without errors but then it doesn't work in a browser throwing this error at me:
Uncaught TypeError: (0 , _jsxDevRuntime).jsxDEV is not a function
On the internet I'm finding only babel-related similar errors, but nothing around swc.

![react_swc_issue](https://user-images.githubusercontent.com/577112/159943435-a2ac9909-1536-43f1-9e82-7a153bb19ade.png)

Any ideas?

I'll add that the app has mixed .tsx and .jsx files (we are in the middle of migration to TS).
Also the app is being managed by the [NX](https://nx.dev/) tool

**Comment by RiESAEX** [comment]
can you share a repro?

**Comment by kdy1** [comment]
Use classic runtime or update react

**Reply by KKrisu** [reply]
I have react 17.02 and have tried both runtime settings.

**Comment by KKrisu** [comment, marked as answer]
In my case, it was a missing "defaultConfiguration" option for "build" in workspace.json. After setting it to "production" it's working fine.
